packages/seacas/libraries/ioss/src/exodus/Ioex_ParallelDatabaseIO.C

```diff
@@ -553,7 +556,7 @@ namespace Ioex {
 
 #if !defined(__IOSS_WINDOWS__)
     if (!path.empty()) {
-      chdir(current_cwd);
+      (void)chdir(current_cwd);
```

Comment:
That probably quiets the warning on some platforms, but I should probably actually do some error checking to make sure the chdir worked...